lib/view/dailyEventView.dart

```diff
@@ -153,19 +143,19 @@ class _SingleDayState extends State<SingleDay> {
       }
     }
     double space = width / eventsInHour.length;
-    //print("hour is $hour");
-    //print("events starting in hour: $eventsInHour");
-    List<CustomPaint> eventsToPaint = eventsInHour
+    List<GestureDetector> eventsToPaint = eventsInHour
         .map(
-          (event) => CustomPaint(
-            painter: MyPainter(context, eventSpace: space, event: event),
-            child: GestureDetector(
-                behavior: HitTestBehavior.translucent,
-                onTap: () {
-                  showEventDetailPopup(context, event, widget.date);
-                },
-                child: SizedBox(
-                    width: space, child: const Card(color: Colors.black))),
+          (event) => GestureDetector(
+            onTap: () {
+              showEventDetailPopup(context, event, widget.date);
+            },
+            child: CustomPaint(
+              painter: MyPainter(context, eventWidth: space, event: event),
+              child: Container(
+                width: space,
+                height: hourHeight,
```

Comment:
I think GestureDetector expands to fit all of its children. Maybe [something like this](https://stackoverflow.com/a/74403333) could be done as quick fix before the release?